pygmt/datasets/load_remote_dataset.py

```diff
@@ -227,6 +227,29 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution(["gridline"], True),
         },
     ),
+    "moon_relief": GMTRemoteDataset(
+        title="Moon relief",
+        name="moon relief",
+        long_name="Moon relief",
+        units="meters",
+        extra_attributes={},
```

Comment:
Please note that `extra_attributes` is empty here because I have no idea about the `horizontal_datum` attribute for the dataset.